app/models/contestant.rb

```diff
@@ -1,3 +1,11 @@
 class Contestant < ApplicationRecord
+  has_many :contestant_projects 
+  has_many :projects, through: :contestant_projects 
 
+  # def projects
+  #   binding.pry
+  #   self.projects.map do |project|
+  #     project.name
+  #   end
+  # end
```

Comment:
I know these are commented out, so don't want to spend too much time here, just remember that because you've set up the relationships on lines 2 and 3, you now have methods like `contestant1.projects` which will return to you all the project objects for that contestant. If you wanted just the project names, do you remember the active record method that will pull just the values from a column? Look into `.pluck`